tests/func/machine/test_machine_config.py

```diff
@@ -0,0 +1,115 @@
+import textwrap
+
+import pytest
+
+from dvc.main import main
+
+config_text = textwrap.dedent(
```

Comment:
Consider capitalizing these things (and the other constants added in this PR) to make it more obvious that it's fixed constant data and not a test specific variable or fixture

```suggestion
CONFIG_TEXT = textwrap.dedent(
```